socialhome/streams/app/tests/components/StreamElement.tests.js

```diff
@@ -12,6 +12,20 @@ describe("StreamElement", () => {
         Sinon.restore()
     })
 
+    describe("template", () => {
+        describe("share reaction", () => {
+            it("has item-reaction-shared if has shared", () => {
+                let propsData = getStreamElementPropsData({hasShared: false})
+                let target = mount(StreamElement, {propsData})
+                target.find(".item-reaction.item-reaction-shared").length.should.eq(0)
```

Comment:
`eq`? I don't see that in Chai's documentation. Where does it come from?

Reply:
http://chaijs.com/api/bdd/#method_equal - it's a shorter nicer alias for `equal`.

Reply:
Ah, yes. I really don't understand this mania of having 36 ways of doing one thing...